editor/src/index.js

```diff
@@ -29,4 +31,8 @@ if ( parseInt( tiTpc.tier ) === 3 ) {
 		render: Exporter,
 		icon,
 	} );
+
+	registerPlugin( 'ti-tpc-templates-button', {
+		render: renderWrappedButton,
+	} );
```

Comment:
Given Templates Patterns block import method is available to all users, we can also make it available to all users and not restrict it for Pro-only.